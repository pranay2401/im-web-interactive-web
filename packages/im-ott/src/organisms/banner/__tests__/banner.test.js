import React from "react";
import { cleanup } from "@testing-library/react";
import Banner from "../Banner";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("<Banner/> should render correctly", () => {
  const tree = renderer.create(<Banner />).toJSON();
  expect(tree).toMatchSnapshot();
});
