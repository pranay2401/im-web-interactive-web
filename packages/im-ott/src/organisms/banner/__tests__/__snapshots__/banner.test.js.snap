// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner should render correctly 1`] = `
<div
  className="ant-carousel"
>
  <div
    className="slick-slider slick-initialized"
    dir="ltr"
  >
    <div
      className="slick-list"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={null}
      onMouseMove={null}
      onMouseUp={[Function]}
      onTouchCancel={null}
      onTouchEnd={[Function]}
      onTouchMove={null}
      onTouchStart={[Function]}
      style={Object {}}
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "WebkitTransform": "translate3d(0px, 0px, 0px)",
            "WebkitTransition": "",
            "msTransform": "translateX(0px)",
            "opacity": 1,
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      >
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={-1}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <div
                className="ant-image"
                style={
                  Object {
                    "height": undefined,
                    "width": "100%",
                  }
                }
              >
                <img
                  className="ant-image-img bannerImage"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="Banner-image-2.jpg"
                  style={
                    Object {
                      "height": undefined,
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <div
                className="ant-image"
                style={
                  Object {
                    "height": undefined,
                    "width": "100%",
                  }
                }
              >
                <img
                  className="ant-image-img bannerImage"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="Banner-image-1.jpg"
                  style={
                    Object {
                      "height": undefined,
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <div
                className="ant-image"
                style={
                  Object {
                    "height": undefined,
                    "width": "100%",
                  }
                }
              >
                <img
                  className="ant-image-img bannerImage"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="Banner-image-2.jpg"
                  style={
                    Object {
                      "height": undefined,
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <div
                className="ant-image"
                style={
                  Object {
                    "height": undefined,
                    "width": "100%",
                  }
                }
              >
                <img
                  className="ant-image-img bannerImage"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="Banner-image-1.jpg"
                  style={
                    Object {
                      "height": undefined,
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={3}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <div
                className="ant-image"
                style={
                  Object {
                    "height": undefined,
                    "width": "100%",
                  }
                }
              >
                <img
                  className="ant-image-img bannerImage"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="Banner-image-2.jpg"
                  style={
                    Object {
                      "height": undefined,
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <ul
      className="slick-dots slick-dots-bottom"
      onMouseEnter={null}
      onMouseLeave={null}
      onMouseOver={null}
      style={
        Object {
          "display": "block",
        }
      }
    >
      <li
        className="slick-active"
      >
        <button
          onClick={[Function]}
        >
          1
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          2
        </button>
      </li>
    </ul>
  </div>
</div>
`;
